Runtime/Exporters/CubemapExporter.cs

```diff
@@ -108,7 +108,7 @@ private void ExportCubemap(ImageQuality imageQuality)
                     {
                         throw new System.NotSupportedException("Current graphic device/platform does not support RenderToCubemap.");
                     }
-                    var path = Directory.CreateDirectory(Path.Combine(Map.exportSettings.OutputDirectory, "images", exportAllQualities ? imageQuality.ToString().ToLower() : "", current.Id).ToOSSpecificPath());
+                    var path = Directory.CreateDirectory(Path.Combine(Map.projectFolder, ".treasured", Map.projectFolder, "images", exportAllQualities ? imageQuality.ToString().ToLower() : "", current.Id).ToOSSpecificPath());
```

Comment:
Should be using Map.exportSettings.OutputDirectory to make it backward compatible